xla/service/hlo_runner_interface.h

```diff
@@ -214,11 +214,15 @@ class HloRunnerInterface {
   static absl::StatusOr<std::unique_ptr<HloModule>> CreateModuleFromString(
       absl::string_view hlo_string, const DebugOptions& debug_options);
 
+  // Creates an HloModule from the given proto.
+  static absl::StatusOr<std::unique_ptr<HloModule>> HloProtoToModule(
+      const HloModuleProto& proto, const DebugOptions& = {});
+
   // Reads the proto file in xla.HloProto format, creates and returns the
   // HloModule.
   static absl::StatusOr<std::unique_ptr<HloModule>>
-  ReadModuleFromBinaryProtoFile(const std::string& filename,
-                                const DebugOptions& debug_options);
+  ReadModuleFromBinaryProtoFile(absl::string_view filename,
+                                const DebugOptions& debug_options = {});
```

Comment:
To avoid creating a temporary default value:
```suggestion
                                const DebugOptions& debug_options = DebugOptions::default_instance());
```